### PR TITLE
renamed 'config' to 'config_filepath'

### PR DESCRIPTION
**Pull request by cardinalion**

### CODE REVIEW

**Comment by pylipp** [comment]
Thanks again :) can you check the travis output about why the tests fail? I suspect it's due to a docstring line being too long.

**Comment by cardinalion** [comment]
I got an error from flake8 saying F821 undefined name 'params'
from the line:
client = Client(
        configuration=configuration, out=Client.Out(logger.info, logger.error))
    success, store_offline = client.safely_run(command, **params)

and

if store_offline and offline.add(command, **params):
        logger.info("Stored '{}' request in offline backup.".format(command))

I'm not sure why

**Comment by cardinalion** [comment]
Same, the pre-commit test is good but the continuous-integration failed. I'm not sure how to solve it

File: financeager/cli.py [hunk 1]
```diff
@@ -35,16 +35,15 @@ def main():
...
-    """High-level API entry point.
-    All 'params' are passed to 'Client.safely_run()'.
-    'config' specifies the path to a custom config file (optional). If 'verbose'
-    is set, debug level log messages are printed to the terminal.
-
-    This function can be used for scripting. Provide 'command' and 'params'
-    according to what the command line interface accepts (consult help via
-    `financeager [command] --help`), e.g. {"command": "add", "name":
-    "champagne", "value": "99"}.
+def run(command=None, config_filepath=None, verbose=False, **cl_kwargs):
```
Comment (pylipp) [inline financeager/cli.py]: It should be `params` instead of `cl_kwargs`, I recently changed it on master

File: financeager/cli.py [hunk 2]
```diff
@@ -35,16 +35,15 @@ def main():
...
+    """High-level API entry point, useful for scripts.
+    Run 'command' passing 'cl_kwargs' according to
+    what the command line interface accepts
+    (consult help via `financeager [command] --help`),
+    e.g. {"command": "add", "name":"champagne", "value": "99"}.
+    All kwargs are passed to 'communication.run()'.
+    'config_filepath' specifies the path to a custom config file (optional).
+    If 'verbose'is set, debug level log messages are printed to the terminal.
```
Comment (pylipp) [inline financeager/cli.py]: Can you re-check this for spaces? E.g. between name and champagne, and verbose and is.

File: financeager/cli.py [hunk 3]
```diff
@@ -35,16 +35,15 @@ def main():
...
-    "champagne", "value": "99"}.
+def run(command=None, config_filepath=None, verbose=False, **params):
+    """High-level API entry point, useful for scripts.
+    Run 'command' passing 'params' according to
+    what the command line interface accepts
+    (consult help via `financeager [command] --help`),
+    e.g. {"command": "add", "name": "champagne", "value": "99"}.
+    All kwargs are passed to 'communication.run()'.
+    'config_filepath' specifies the path to a custom config file (optional).
+    If 'verbose' is set, debug level log messages are printed to the terminal.
```
Comment (pylipp) [inline financeager/cli.py]: Sorry for noticing so late. The docstring should be identical to the one from the master. Can you update it once more?
You can also grant me permission to contribute to the pull-request from inside your PR settings if you want :)

**Comment by pylipp** [comment]
Thanks again! Almost there now :)

**Comment by cardinalion** [comment]
Anything I could do now? I think you can edit it.

**Comment by pylipp** [comment]
Thanks again, and congratz :+1:

**Pull request merged**